client/dhrystone.cpp

```diff
@@ -90,7 +90,7 @@ int dhrystone(
     unsigned long         Loops;
     DS_DATA dd;
 
-    double                  startclock, endclock;
```

Comment:
This is not related to this PR.
Please create a separate branch and a separate PR with this change.

Reply:
Reverted back and resubmitted the change via a separate PR.